src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,240 @@
+package duke.parser;
```

Comment:
This class is way too long, it would be better if you can divide it into classes. Also, the parser should only parse the input to command rather than creating task object(todo, deadline etc). Such classes can be created in AddCommand. It would be more OOP in this way.